nomos-services/data-availability/verifier/src/storage/adapters/rocksdb.rs

```diff
@@ -109,16 +132,6 @@ where
     }
 }
 
-// Combines a 32-byte blob ID (`[u8; 32]`) with a 2-byte column index
-// (`u16` represented as `[u8; 2]`).
-fn create_blob_idx(blob_id: &[u8], column_idx: &[u8]) -> [u8; 34] {
-    let mut blob_idx = [0u8; 34];
-    blob_idx[..blob_id.len()].copy_from_slice(blob_id);
-    blob_idx[blob_id.len()..].copy_from_slice(column_idx);
-
-    blob_idx
-}
-
```

Comment:
Why did we had this so duplicated everywhere. Any hint @bacv ?

Reply:
Because it is related to the rocks db adapters only, didn't seem to fit in `nomos-da` core. To have rocksdb specific code in `kzgrs-backend` isn't also ideal, but better for managing. @andrussal maybe lets add a comment about usage of `create_blob_idx` or a more explicit name?

Reply:
> Because it is related to the rocks db adapters only, didn't seem to fit in `nomos-da` core. To have rocksdb specific code in `kzgrs-backend` isn't also ideal, but better for managing. @andrussal maybe lets add a comment about usage of `create_blob_idx` or a more explicit name?

We have a utils crate maybe can be moved there?
Also, @bacv, this is just for computing the key of the query right? Because in this pr is being use for the `blob_idx` which should be related to the subnetwork/column iirc?